python/fbprophet/forecaster.py

```diff
@@ -98,7 +94,7 @@ def __init__(
             mcmc_samples=0,
             interval_width=0.80,
             uncertainty_samples=1000,
-            stan_backend='pystan'
+            stan_backend=StanBackendEnum.PYSTAN
```

Comment:
Perhaps the better default here is None, and then if None we can select the first available backend from
```
get_backends_from_env()[0]
```
Then if you only have cmdstanpy backend setup, it will work without having to specify it here.
That would also clean up all of the edits to the tests below since they would work without having to specify the backend.

Reply:
The original intention behind that env var is to serve only as a command switch for installation (setup.py).
How about we stored the default value in a file along with models' binaries instead and thus make the runtime more hermetical?

I would not users' code depend on some hidden env variable.

Reply:
Yeah that sounds great to require setting the env variable for cmdstanpy only for install. Ideally after that it would work seamlessly the same as with pystan without having to specify the cmdstanpy backend on model init. What do you think about a strategy where (if no backend is specified) here in __init__ we could try loading both of them, and then if only one is available (catch the IOError when it tries to load the model file) that is the one we use. If both are installed, we could default to pystan. With this approach, if you have only cmdstanpy set up, then it will use cmdstanpy without having to do any extra work. You would only have to specify cmdstanpy if you have both it and pystan installed (and built with fbprophet).